client/components/cloud/project/billing/additional-services/additional-services.component.js

```diff
@@ -0,0 +1,10 @@
+const AdditionalServicesComponent = {
```

Comment:
/cc @cbourgois which is actually migrating `client/components/cloud/*` to `client/app/cloud/components`.